INFO 2023-11-11 00:53:22,281 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 00:53:22,281 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 00:53:28,700 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 00:53:28,700 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 00:53:28,714 playlist_service playlist_service.py 45 Error when trying to create the playlist
INFO 2023-11-11 00:53:28,714 playlist_service playlist_service.py 45 Error when trying to create the playlist
INFO 2023-11-11 00:53:28,714 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,714 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,714 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,714 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,715 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,715 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,715 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,715 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,715 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,715 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,715 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,715 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,716 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,716 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,716 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,716 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,717 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,717 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,717 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,717 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,718 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,718 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,718 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,718 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,718 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,718 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,719 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,719 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,719 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,719 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,719 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,719 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,720 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,720 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,720 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,720 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,720 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,720 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,720 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,720 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,721 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,721 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,721 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,721 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,721 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,721 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,721 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,721 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,722 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,722 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,722 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,722 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,722 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,722 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,722 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,722 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,723 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,723 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,723 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,723 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,723 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,723 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,723 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,723 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,724 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,724 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,724 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,724 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,724 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,724 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,724 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,724 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,725 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,725 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,725 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,725 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,725 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,725 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,726 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,726 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,726 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,726 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,726 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,726 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,726 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,726 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,726 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,726 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,727 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,727 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,727 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,727 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,727 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,727 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,727 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,727 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,728 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,728 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,728 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,728 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,728 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,728 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,728 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,728 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,729 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,729 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,729 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,729 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,729 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,729 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,729 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,729 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,730 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,730 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,730 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,730 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,730 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,730 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,730 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,730 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,731 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,731 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,731 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,731 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,732 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,732 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,732 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,732 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,732 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,732 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,732 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,732 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,733 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,733 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,733 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,733 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,733 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,733 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,733 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,733 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,734 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,734 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,734 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,734 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,734 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,734 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,734 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,734 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,735 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,735 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,735 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,735 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,735 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,735 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,735 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,735 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,735 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,735 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,736 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,736 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,736 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,736 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,736 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,736 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,736 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,736 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,736 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,736 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,737 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,737 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,737 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,737 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,737 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,737 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,737 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,737 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,738 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,738 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,738 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,738 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,738 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,738 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,738 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,738 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,739 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,739 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,739 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,739 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,739 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,739 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,739 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,739 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,740 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,740 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,740 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,740 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,740 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,740 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,740 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,740 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,740 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,740 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,740 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,740 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,741 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,741 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,741 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,741 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,741 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,741 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,741 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,741 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,742 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,742 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,742 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,742 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,742 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,742 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,743 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,743 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,743 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,743 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,743 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,743 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,743 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,743 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,743 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,743 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,744 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,744 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,744 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,744 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,744 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,744 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,744 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,744 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,745 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,745 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,745 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,745 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,745 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,745 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,745 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,745 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,746 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,746 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,746 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,746 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,746 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,746 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,746 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,746 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,747 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,747 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,747 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,747 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,747 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,747 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,747 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,747 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,747 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,747 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,748 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,748 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,748 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,748 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,748 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,748 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,748 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,748 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,748 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,748 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,749 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,749 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,749 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,749 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,749 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,749 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,749 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,749 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,750 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,750 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,750 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,750 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,750 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,750 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,750 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,750 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,751 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,751 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,751 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,751 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,751 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,751 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,751 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,751 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,752 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,752 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,752 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,752 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,752 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,752 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,752 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,752 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,752 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,752 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,753 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,753 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,753 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,753 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,753 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,753 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,753 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,753 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,753 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,753 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,754 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,754 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,754 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,754 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,754 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,754 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,754 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,754 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,755 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,755 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,755 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,755 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,755 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,755 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,755 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,755 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,755 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,755 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,756 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,756 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,756 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,756 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,756 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,756 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,756 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,756 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,756 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,756 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,757 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,757 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,757 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,757 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,757 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,757 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,758 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,758 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,758 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,758 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,758 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,758 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,758 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,758 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,759 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,759 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,759 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,759 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,759 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,759 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,759 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,759 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,759 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,759 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,760 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,760 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,760 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,760 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,760 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,760 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,760 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,760 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,760 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,760 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,761 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,761 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,761 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,761 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,761 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,761 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,761 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,761 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,762 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,762 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,762 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,762 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,762 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,762 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,762 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,762 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,762 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,762 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,763 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,763 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,763 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,763 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,763 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,763 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,763 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,763 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,764 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,764 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,764 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,764 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,764 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,764 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,764 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,764 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,765 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,765 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,765 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,765 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,765 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,765 playlist_service playlist_service.py 61 An exception occured in get_playlist:
INFO 2023-11-11 00:53:28,765 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
INFO 2023-11-11 00:53:28,765 playlist_service playlist_service.py 99 tried getting playlist when adding track to it                                  but DNE in db yet. Check for race conditions.                                 Caller needs to check if playlist exsits,                                  then add it, not this function's job
ERROR 2023-11-11 00:53:29,428 log log.py 241 Internal Server Error: /spotify/users/franserr/playlists
Traceback (most recent call last):
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django_injector/apps.py", line 169, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/utils/decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/views/spotify/spotify_playlists_view.py", line 34, in post
    persist_retrived_data(services, parsedInfo)
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/persistence_util.py", line 47, in persist_retrived_data
    persistence.image.link_images_to_artists(artists)
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/spotify/persistence_service.py", line 99, in link_images_to_artists
    self.link_image_to_artist(image, artist_uri)
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/spotify/persistence_service.py", line 85, in link_image_to_artist
    self.image_serivce.link_image_to_artist(artist_uri, image_url)
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/core/image_service.py", line 80, in link_image_to_artist
    image.save()
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/models.py", line 105, in save
    self.clean()
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/models.py", line 96, in clean
    raise ValidationError(
django.core.exceptions.ValidationError: ['Image cannot be linked to an artist && album.']
ERROR 2023-11-11 00:53:29,428 log log.py 241 Internal Server Error: /spotify/users/franserr/playlists
Traceback (most recent call last):
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django_injector/apps.py", line 169, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/utils/decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/views/spotify/spotify_playlists_view.py", line 34, in post
    persist_retrived_data(services, parsedInfo)
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/persistence_util.py", line 47, in persist_retrived_data
    persistence.image.link_images_to_artists(artists)
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/spotify/persistence_service.py", line 99, in link_images_to_artists
    self.link_image_to_artist(image, artist_uri)
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/spotify/persistence_service.py", line 85, in link_image_to_artist
    self.image_serivce.link_image_to_artist(artist_uri, image_url)
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/core/image_service.py", line 80, in link_image_to_artist
    image.save()
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/models.py", line 105, in save
    self.clean()
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/models.py", line 96, in clean
    raise ValidationError(
django.core.exceptions.ValidationError: ['Image cannot be linked to an artist && album.']
INFO 2023-11-11 01:00:00,969 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/dtos/retrieval_dtos.py changed, reloading.
INFO 2023-11-11 01:00:00,969 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/dtos/retrieval_dtos.py changed, reloading.
INFO 2023-11-11 01:00:01,485 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 01:00:01,485 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 01:01:06,148 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/parse_results_util.py changed, reloading.
INFO 2023-11-11 01:01:06,148 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/parse_results_util.py changed, reloading.
INFO 2023-11-11 01:01:06,663 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 01:01:06,663 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 01:02:46,184 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/parse_results_util.py changed, reloading.
INFO 2023-11-11 01:02:46,184 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/parse_results_util.py changed, reloading.
INFO 2023-11-11 01:02:46,774 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 01:02:46,774 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 01:03:35,036 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/parse_results_util.py changed, reloading.
INFO 2023-11-11 01:03:35,036 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/parse_results_util.py changed, reloading.
INFO 2023-11-11 01:03:35,672 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 01:03:35,672 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 01:03:43,050 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 01:03:43,050 album_service album_service.py 42 An error occurred while creating an album:
ERROR 2023-11-11 01:03:43,925 log log.py 241 Internal Server Error: /spotify/users/franserr/playlists
Traceback (most recent call last):
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django_injector/apps.py", line 169, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/utils/decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/views/spotify/spotify_playlists_view.py", line 34, in post
    persist_retrived_data(services, parsedInfo)
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/persistence_util.py", line 47, in persist_retrived_data
    persistence.image.link_images_to_artists(artists)
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/spotify/persistence_service.py", line 99, in link_images_to_artists
    self.link_image_to_artist(image, artist_uri)
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/spotify/persistence_service.py", line 85, in link_image_to_artist
    self.image_serivce.link_image_to_artist(artist_uri, image_url)
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/core/image_service.py", line 80, in link_image_to_artist
    image.save()
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/models.py", line 105, in save
    self.clean()
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/models.py", line 96, in clean
    raise ValidationError(
django.core.exceptions.ValidationError: ['Image cannot be linked to an artist && album.']
ERROR 2023-11-11 01:03:43,925 log log.py 241 Internal Server Error: /spotify/users/franserr/playlists
Traceback (most recent call last):
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django_injector/apps.py", line 169, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/utils/decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/views/spotify/spotify_playlists_view.py", line 34, in post
    persist_retrived_data(services, parsedInfo)
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/persistence_util.py", line 47, in persist_retrived_data
    persistence.image.link_images_to_artists(artists)
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/spotify/persistence_service.py", line 99, in link_images_to_artists
    self.link_image_to_artist(image, artist_uri)
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/spotify/persistence_service.py", line 85, in link_image_to_artist
    self.image_serivce.link_image_to_artist(artist_uri, image_url)
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/core/image_service.py", line 80, in link_image_to_artist
    image.save()
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/models.py", line 105, in save
    self.clean()
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/models.py", line 96, in clean
    raise ValidationError(
django.core.exceptions.ValidationError: ['Image cannot be linked to an artist && album.']
INFO 2023-11-11 01:04:41,639 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/models.py changed, reloading.
INFO 2023-11-11 01:04:41,639 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/models.py changed, reloading.
INFO 2023-11-11 01:04:42,810 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 01:04:42,810 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 01:05:00,332 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 01:05:00,332 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 01:05:06,459 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 01:05:06,459 album_service album_service.py 42 An error occurred while creating an album:
WARNING 2023-11-11 01:06:28,113 log log.py 241 Not Found: /img/icon-changelink.svg
WARNING 2023-11-11 01:06:28,113 log log.py 241 Not Found: /img/icon-changelink.svg
WARNING 2023-11-11 01:06:28,115 log log.py 241 Not Found: /img/icon-addlink.svg
WARNING 2023-11-11 01:06:28,115 log log.py 241 Not Found: /img/icon-addlink.svg
WARNING 2023-11-11 01:06:30,564 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 01:06:30,564 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 01:06:30,566 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 01:06:30,566 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 01:29:36,524 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 01:29:36,524 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 01:29:36,525 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 01:29:36,525 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 01:30:12,575 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 01:30:12,575 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 01:30:12,576 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 01:30:12,576 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 01:30:17,569 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 01:30:17,569 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 01:30:17,571 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 01:30:17,571 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
INFO 2023-11-11 04:06:32,711 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:06:32,711 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:06:53,432 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/spotify/retrieval/spotify_playlist_service.py changed, reloading.
INFO 2023-11-11 04:06:53,432 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/spotify/retrieval/spotify_playlist_service.py changed, reloading.
INFO 2023-11-11 04:06:53,958 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:06:53,958 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:07:17,926 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,926 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,928 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,928 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,930 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,930 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,931 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,931 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,932 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,932 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,934 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,934 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,935 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,935 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,937 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,937 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,938 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,938 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,939 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,939 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,941 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,941 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,942 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,942 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,944 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,944 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,945 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,945 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,947 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,947 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,948 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,948 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,950 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,950 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,952 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,952 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,953 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,953 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,955 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,955 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,956 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,956 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,958 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,958 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,959 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,959 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,961 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,961 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,963 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,963 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,964 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,964 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,966 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,966 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,967 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,967 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,968 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,968 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,969 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,969 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,971 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,971 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,972 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,972 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,974 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,974 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,975 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,975 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,977 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,977 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,978 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,978 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,979 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,979 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,981 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,981 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,982 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,982 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,983 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,983 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,985 user_service user_service.py 65 User does not exist
INFO 2023-11-11 04:07:17,985 user_service user_service.py 65 User does not exist
WARNING 2023-11-11 04:09:41,768 log log.py 241 Not Found: /admin/img/icon-changelink.svg
WARNING 2023-11-11 04:09:41,768 log log.py 241 Not Found: /admin/img/icon-changelink.svg
WARNING 2023-11-11 04:09:41,769 log log.py 241 Not Found: /admin/img/icon-addlink.svg
WARNING 2023-11-11 04:09:41,769 log log.py 241 Not Found: /admin/img/icon-addlink.svg
WARNING 2023-11-11 04:09:45,168 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 04:09:45,168 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 04:09:45,170 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 04:09:45,170 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 04:09:45,187 log log.py 241 Not Found: /favicon.ico
WARNING 2023-11-11 04:09:45,187 log log.py 241 Not Found: /favicon.ico
WARNING 2023-11-11 04:10:03,968 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 04:10:03,968 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 04:10:03,969 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 04:10:03,969 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
INFO 2023-11-11 04:11:59,495 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:11:59,495 autoreload autoreload.py 668 Watching for file changes with StatReloader
WARNING 2023-11-11 04:12:01,268 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 04:12:01,268 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 04:12:01,269 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 04:12:01,269 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
INFO 2023-11-11 04:15:19,067 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/core/user_service.py changed, reloading.
INFO 2023-11-11 04:15:19,067 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/core/user_service.py changed, reloading.
INFO 2023-11-11 04:15:19,590 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:15:19,590 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:15:21,813 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/core/user_service.py changed, reloading.
INFO 2023-11-11 04:15:21,813 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/core/user_service.py changed, reloading.
INFO 2023-11-11 04:15:22,357 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:15:22,357 autoreload autoreload.py 668 Watching for file changes with StatReloader
WARNING 2023-11-11 04:16:25,073 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 04:16:25,073 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 04:16:25,075 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 04:16:25,075 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 04:16:27,664 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 04:16:27,664 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 04:16:27,667 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 04:16:27,667 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 04:16:34,705 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 04:16:34,705 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 04:16:34,707 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 04:16:34,707 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
INFO 2023-11-11 04:17:20,432 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/persistence_util.py changed, reloading.
INFO 2023-11-11 04:17:20,432 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/persistence_util.py changed, reloading.
INFO 2023-11-11 04:17:20,958 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:17:20,958 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:20:05,080 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/views/spotify/spotify_playlists_view.py changed, reloading.
INFO 2023-11-11 04:20:05,080 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/views/spotify/spotify_playlists_view.py changed, reloading.
INFO 2023-11-11 04:20:05,736 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:20:05,736 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:22:22,661 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/spotify/retrieval/spotify_playlist_service.py changed, reloading.
INFO 2023-11-11 04:22:22,661 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/spotify/retrieval/spotify_playlist_service.py changed, reloading.
INFO 2023-11-11 04:22:23,181 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:22:23,181 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:24:19,367 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/persistence_util.py changed, reloading.
INFO 2023-11-11 04:24:19,367 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/persistence_util.py changed, reloading.
INFO 2023-11-11 04:24:19,894 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:24:19,894 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:24:56,004 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/spotify/retrieval/spotify_playlist_service.py changed, reloading.
INFO 2023-11-11 04:24:56,004 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/spotify/retrieval/spotify_playlist_service.py changed, reloading.
INFO 2023-11-11 04:24:56,499 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:24:56,499 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:35:10,465 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/spotify/retrieval/spotify_playlist_service.py changed, reloading.
INFO 2023-11-11 04:35:10,465 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/spotify/retrieval/spotify_playlist_service.py changed, reloading.
INFO 2023-11-11 04:35:11,076 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:35:11,076 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:35:13,309 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/spotify/retrieval/spotify_playlist_service.py changed, reloading.
INFO 2023-11-11 04:35:13,309 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/spotify/retrieval/spotify_playlist_service.py changed, reloading.
INFO 2023-11-11 04:35:13,834 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:35:13,834 autoreload autoreload.py 668 Watching for file changes with StatReloader
WARNING 2023-11-11 04:35:15,431 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 04:35:15,432 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 04:35:15,432 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 04:35:15,431 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
INFO 2023-11-11 04:39:26,438 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/parse_results_util.py changed, reloading.
INFO 2023-11-11 04:39:26,438 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/parse_results_util.py changed, reloading.
INFO 2023-11-11 04:39:26,933 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:39:26,933 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:39:32,234 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/spotify/retrieval/spotify_playlist_service.py changed, reloading.
INFO 2023-11-11 04:39:32,234 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/spotify/retrieval/spotify_playlist_service.py changed, reloading.
INFO 2023-11-11 04:39:32,748 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:39:32,748 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:41:35,610 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/parse_results_util.py changed, reloading.
INFO 2023-11-11 04:41:35,610 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/parse_results_util.py changed, reloading.
INFO 2023-11-11 04:41:36,125 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:41:36,125 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:42:14,270 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/parse_results_util.py changed, reloading.
INFO 2023-11-11 04:42:14,270 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/parse_results_util.py changed, reloading.
INFO 2023-11-11 04:42:14,799 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:42:14,799 autoreload autoreload.py 668 Watching for file changes with StatReloader
WARNING 2023-11-11 04:45:18,371 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 04:45:18,371 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 04:45:18,372 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 04:45:18,372 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 04:45:20,570 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 04:45:20,570 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 04:45:20,571 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 04:45:20,571 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
INFO 2023-11-11 04:46:24,599 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/parse_results_util.py changed, reloading.
INFO 2023-11-11 04:46:24,599 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/parse_results_util.py changed, reloading.
INFO 2023-11-11 04:46:25,124 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:46:25,124 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:46:49,967 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/spotify/retrieval/spotify_playlist_service.py changed, reloading.
INFO 2023-11-11 04:46:49,967 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/spotify/retrieval/spotify_playlist_service.py changed, reloading.
INFO 2023-11-11 04:46:50,476 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:46:50,476 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:48:26,466 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/parse_results_util.py changed, reloading.
INFO 2023-11-11 04:48:26,466 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/parse_results_util.py changed, reloading.
INFO 2023-11-11 04:48:27,062 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:48:27,062 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:50:04,428 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/parse_results_util.py changed, reloading.
INFO 2023-11-11 04:50:04,428 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/parse_results_util.py changed, reloading.
INFO 2023-11-11 04:50:05,070 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:50:05,070 autoreload autoreload.py 668 Watching for file changes with StatReloader
ERROR 2023-11-11 04:50:22,441 log log.py 241 Internal Server Error: /spotify/users/franserr/playlists
Traceback (most recent call last):
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django_injector/apps.py", line 169, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/utils/decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/views/spotify/spotify_playlists_view.py", line 33, in post
    parsedInfo = sp_playlist_service.get_user_liked_playlists()
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/spotify/retrieval/spotify_playlist_service.py", line 25, in get_user_liked_playlists
    info = self._begin_build(
           ^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/spotify/retrieval/spotify_playlist_service.py", line 65, in _begin_build
    info = get_tracks(
           ^^^^^^^^^^^
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/parse_results_util.py", line 210, in get_tracks
    info, missing = process_source(
                    ^^^^^^^^^^^^^^^
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/parse_results_util.py", line 261, in process_source
    more_missing, more_track_uris = paginate_results(
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: cannot unpack non-iterable NoneType object
ERROR 2023-11-11 04:50:22,441 log log.py 241 Internal Server Error: /spotify/users/franserr/playlists
Traceback (most recent call last):
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django_injector/apps.py", line 169, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/utils/decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/Users/franserr/opt/anaconda3/envs/tracker/lib/python3.12/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/views/spotify/spotify_playlists_view.py", line 33, in post
    parsedInfo = sp_playlist_service.get_user_liked_playlists()
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/spotify/retrieval/spotify_playlist_service.py", line 25, in get_user_liked_playlists
    info = self._begin_build(
           ^^^^^^^^^^^^^^^^^^
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/spotify/retrieval/spotify_playlist_service.py", line 65, in _begin_build
    info = get_tracks(
           ^^^^^^^^^^^
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/parse_results_util.py", line 210, in get_tracks
    info, missing = process_source(
                    ^^^^^^^^^^^^^^^
  File "/Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/parse_results_util.py", line 261, in process_source
    more_missing, more_track_uris = paginate_results(
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: cannot unpack non-iterable NoneType object
INFO 2023-11-11 04:53:23,904 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/parse_results_util.py changed, reloading.
INFO 2023-11-11 04:53:23,904 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/parse_results_util.py changed, reloading.
INFO 2023-11-11 04:53:24,415 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:53:24,415 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:56:20,280 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:56:20,280 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-11 04:58:14,657 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:14,657 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:15,037 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:15,037 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:15,052 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:15,052 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:15,072 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:15,072 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:15,074 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:15,074 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:15,084 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:15,084 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:15,091 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:15,091 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:15,105 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:15,105 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:15,217 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:15,217 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:15,242 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:15,242 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:16,840 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:16,840 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:19,749 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:19,749 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:20,846 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:20,846 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:21,333 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:21,333 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:21,456 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:21,456 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:23,078 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:23,078 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:23,094 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:23,094 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:23,099 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:23,099 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:23,104 album_service album_service.py 42 An error occurred while creating an album:
INFO 2023-11-11 04:58:23,104 album_service album_service.py 42 An error occurred while creating an album:
WARNING 2023-11-11 05:00:54,839 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 05:00:54,840 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 05:00:54,840 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 05:00:54,839 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 05:01:07,502 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 05:01:07,502 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 05:01:07,503 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 05:01:07,503 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 05:04:42,944 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 05:04:42,944 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 05:04:42,946 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 05:04:42,946 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 05:05:09,883 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 05:05:09,883 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 05:05:09,885 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 05:05:09,885 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 05:05:45,233 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 05:05:45,233 log log.py 241 Not Found: /admin/spotify_analyzer/img/icon-addlink.svg
WARNING 2023-11-11 05:05:45,234 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
WARNING 2023-11-11 05:05:45,234 log log.py 241 Not Found: /admin/spotify_analyzer/img/tooltag-add.svg
INFO 2023-11-12 07:22:36,168 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-12 07:22:36,168 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-12 07:28:53,096 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-12 07:28:53,096 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-12 07:31:35,863 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-12 07:31:35,863 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-12 07:31:54,515 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-12 07:31:54,515 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-12 07:33:25,166 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/core/album_service.py changed, reloading.
INFO 2023-11-12 07:33:25,166 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/services/core/album_service.py changed, reloading.
INFO 2023-11-12 07:33:25,882 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-12 07:33:25,882 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-12 07:33:53,433 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-12 07:33:53,433 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-12 07:34:38,899 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-12 07:34:38,899 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-12 07:47:25,453 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-12 07:47:25,453 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-12 07:47:55,708 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-12 07:47:55,708 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-12 07:48:51,306 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/apps.py changed, reloading.
INFO 2023-11-12 07:48:51,306 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/apps.py changed, reloading.
INFO 2023-11-12 07:48:51,838 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-12 07:48:51,838 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-12 07:49:52,506 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/parse_results_util.py changed, reloading.
INFO 2023-11-12 07:49:52,506 autoreload autoreload.py 266 /Users/franserr/Documents/course-work/port/new-portfolio/tracker/server/spotify_analyzer/util/parse_results_util.py changed, reloading.
INFO 2023-11-12 07:49:53,015 autoreload autoreload.py 668 Watching for file changes with StatReloader
INFO 2023-11-12 07:49:53,015 autoreload autoreload.py 668 Watching for file changes with StatReloader
